hazelcast/src/main/java/com/hazelcast/config/ProbabilisticQuorumConfigBuilder.java

```diff
@@ -16,12 +16,9 @@
 
 package com.hazelcast.config;
 
+import com.hazelcast.internal.cluster.fd.PhiAccrualClusterFailureDetector;
 import com.hazelcast.quorum.impl.ProbabilisticQuorumFunction;
 
-import static com.hazelcast.internal.cluster.fd.PhiAccrualClusterFailureDetector
```

Comment:
When auto-formatted, the import line is longer than 130 chars. This is not good, since you cannot use the auto-formatter without hassle on this class. Since we need two lines even with the shorter constant, it seems to be the better solution to just import the class and don't use a static import on these Hazelcast properties.

Now the class can be auto-formatted again without headaches.